packages/mds-web-sockets/ws-server.ts

```diff
@@ -106,18 +99,16 @@ export const WebSocketServer = async () => {
     TENANT_ID: 'mds'
   })
 
-  const eventProcessor = async (err: Nullable<NatsError>, msg: Msg) => {
-    if (err) logger.error(err)
-    const data = JSON.parse(msg.data)
-    await writeEvent(data)
-  }
-
-  const telemetryProcessor = async (err: Nullable<NatsError>, msg: Msg) => {
-    if (err) logger.error(err)
-    const data = JSON.parse(msg.data)
-    await writeTelemetry(data)
+  const processor = async (err: Nullable<NatsError>, msg: Msg) => {
+    const entity = msg.subject.split('.')?.[1]
+    if (isSupported(entity)) {
+      await pushToClients(entity, JSON.stringify(msg.data))
+      return
+    }
+    logger.error(`Unprocessable entity of type: ${entity} and data: ${JSON.stringify(msg.data)}`)
```

Comment:
Checking for supported entities may not be necessary anymore since we tailor the subscription now @avatarneil ?

Reply:
Yeah, I think we can remove now